src/pipx/commands/environment.py

```diff
@@ -38,7 +38,7 @@ def environment(value: str) -> ExitCode:
         "PIPX_HOME_ALLOW_SPACE": str(paths.ctx.allow_spaces_in_home_path).lower(),
     }
     if value is None:
-        print("Environment variables (set by user):")
+        print("Environment variables (set by user):")  # type: ignore[unreachable]
```

Comment:
`value` is not `Optional`, so in theory it cannot be `None`.

Do we still want to defend against an actual `None`?